src/components/register/register.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { registerWithEmailAndPassword } from '../../utils/auth_utils';
```

Comment:
🚫 **[eslint]** <[import/named](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/named.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parse errors in imported module '../../utils/auth_utils': Identifier 'signInWithGoogle' has already been declared (95:7)